js/map_utils.js

```diff
@@ -31,34 +31,60 @@ var stateStyle = {
   }
 };
   
-function addCentroids(map, countyCentroids) {
+function addCircles() {
   
-  map.selectAll('county-point')
+  // uses globals map, countyCentroids, scaleCircles, activeCategory
+  
+  var circleGroup = map.append('g')
+    .classed('circles', true);
+  circleGroup.selectAll('county-point')
     .data(countyCentroids.features)
     .enter()
     .append('circle')
     .classed('county-point', true)
     .sort(function(a,b) { 
```

Comment:
I wonder if having the data sorted in R would make any kind of difference?

Reply:
Also, sort is happening in `updateCircles`, too. I think we don't need it here if it is down there.